diff -uprN target.org/srcdeps/mrandersonicednrepl/eastwood/v0v3v6/eastwood/util.clj target.new/srcdeps/mrandersonicednrepl/eastwood/v0v3v6/eastwood/util.clj
--- target.org/srcdeps/mrandersonicednrepl/eastwood/v0v3v6/eastwood/util.clj	2019-07-01 00:31:46.000000000 +0900
+++ target.new/srcdeps/mrandersonicednrepl/eastwood/v0v3v6/eastwood/util.clj	2019-07-01 00:35:44.000000000 +0900
@@ -901,7 +901,7 @@ StringWriter."
 
 
 (defn builtin-config-to-resource [name]
-  (io/resource (str "eastwood/config/" name)))
+  (io/resource (str "mrandersonicednrepl/eastwood/v0v3v6/eastwood/config/" name)))
 
 
 (defn init-warning-enable-config [builtin-config-files config-files opt]
