diff -uprN target.org/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/linters/unused.clj target.new/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/linters/unused.clj
--- target.org/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/linters/unused.clj	2019-03-29 09:15:06.980807338 +0900
+++ target.new/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/linters/unused.clj	2019-03-29 09:52:47.661041006 +0900
@@ -1,7 +1,6 @@
 (ns ^{:mranderson/inlined true} mrandersonicednrepl.eastwood.v0v3v5.eastwood.linters.unused
   (:import (java.lang.reflect Method))
   (:require [clojure.set :as set]
-            [mrandersonicednrepl.datajson.v0v2v6.clojure.data :as data]
             [clojure.string :as str]
             [clojure.java.io :as io]
             [mrandersonicednrepl.eastwood.v0v3v5.eastwood.copieddeps.dep10.clojure.tools.reader.edn :as edn]
@@ -574,13 +573,13 @@ discarded inside null: null'."
                              (with-out-str
                                (binding [pp/*print-right-margin* nil]
                                  (pp/pprint (:form stmt)))))))}))
-        
+
         (util/static-call? stmt)
         (let [m (select-keys stmt [:class :method])
               action (warning-unused-static m)]
           (unused-ret-val-lint-result stmt "static method call"
                                       action m location))
-        
+
         (util/invoke-expr? stmt)
         (let [v1 (get-in stmt [:fn :var])
               ;; Special case for apply.  Issue a warning based upon
@@ -675,7 +674,7 @@ discarded inside null: null'."
                    :host-interop "instance method/field access")
                  (first orig-form)
                  sorted-removed-meta-keys)
-         
+
          :else
          (format "Macro invocation of '%s' has metadata with keys %s that are almost certainly ignored."
                  (first orig-form)
diff -uprN target.org/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/util.clj target.new/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/util.clj
--- target.org/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/util.clj	2019-03-29 09:15:07.620804515 +0900
+++ target.new/srcdeps/mrandersonicednrepl/eastwood/v0v3v5/eastwood/util.clj	2019-03-29 09:53:12.197278658 +0900
@@ -901,7 +901,7 @@ StringWriter."
 
 
 (defn builtin-config-to-resource [name]
-  (io/resource (str "eastwood/config/" name)))
+  (io/resource (str "mrandersonicednrepl/eastwood/v0v3v5/eastwood/config/" name)))
 
 
 (defn init-warning-enable-config [builtin-config-files config-files opt]
